package.json

```diff
@@ -99,5 +99,6 @@
 		"lint": "eslint . --ext .js",
 		"lint:fix": "npm run lint -- --fix",
 		"version": "npm run bundle && git add -A bundle"
-	}
+	},
+	"dependencies": {}
```

Comment:
This empty `dependencies` addition appears to be a side effect of running `npx browserslist@latest --update-db`. I do not see it as harmful, so I left it in place. If we'd prefer to remove it, that is fine with me.

Reply:
Sounds good to me to keep it there 👍